helpers.go

```diff
@@ -153,10 +153,17 @@ func goType(pkg string, f *descriptor.FieldDescriptorProto) string {
 	case descriptor.FieldDescriptorProto_TYPE_MESSAGE:
 		if pkg != "" {
 			pkg = pkg + "."
+		} else {
+			pkg = getPackageTypeName(*f.TypeName) + "."
```

Comment:
Why are you doing this ? We need to find another solution, because this breaks the usage with empty package name